conf/broker.conf

```diff
@@ -1181,7 +1181,7 @@ loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold=4
 
 # When calculating new resource usage, the history usage accounts for.
 # It only takes effect in the ThresholdShedder strategy.
-loadBalancerHistoryResourcePercentage=0.9
+loadBalancerHistoryResourcePercentage=90
```

Comment:
You have to consider the case when new version broker using broker.conf in previous version.

Reply:
This change lacks the compatibility. The new version can only use the new configuration file, the old version use the old one.

Reply:
This is a break change. We better not do this.